docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -1,43 +1,75 @@
 # Getting Started with EdgeX Snaps
 
-[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgexfoundry)
+## Introduction
 
+[Snaps](https://snapcraft.io/docs) are application packages that are easy to install and update while being 
+secure, cross‐platform and self-contained.
+Snaps can be installed on any Linux distribution with [snap support](https://snapcraft.io/docs/installing-snapd).
 
-## Introduction
+Snap packages of EdgeX services are published on the [Snap Store](https://snapcraft.io). The list of all EdgeX snaps is available [below](#edgex-snaps).
+
+## EdgeX Snaps
+The following snaps are maintained by the EdgeX working groups:
+
+Platform snap:
+
+- [edgexfoundry](https://snapcraft.io/edgexfoundry): the main platform snap containing
+all reference core services along with several other security, supporting, application, and device services.
+
+Development tools:
+
+- [edgex-ui](https://snapcraft.io/edgex-ui)
+- [edgex-cli](https://snapcraft.io/edgex-cli)
+
+Application services:
+
+- [edgex-app-service-configurable](https://snapcraft.io/edgex-app-service-configurable)
+
+Device services:
+
+- [edgex-device-camera](https://snapcraft.io/edgex-device-camera)
+- [edgex-device-modbus](https://snapcraft.io/edgex-device-modbus)
+- [edgex-device-mqtt](https://snapcraft.io/edgex-device-mqtt)
+- [edgex-device-rest](https://snapcraft.io/edgex-device-rest)
+- [edgex-device-snmp](https://snapcraft.io/edgex-device-snmp)
+- [edgex-device-grove](https://snapcraft.io/edgex-device-grove)
 
-[Snaps](https://snapcraft.io/docs) are a hybrid of traditional Linux packages and containers. Snaps can be installed on any
-Linux distro with snap support enabled, although full confinement currently requires some flavor of Ubuntu (Desktop/Server
-or [Core](https://ubuntu.com/core/docs)).
+Other EdgeX snaps do exist on the public Snap Store ([search by keyword](https://snapcraft.io/search?q=edgex)) or private stores under brand accounts.
 
-Canonical publishes snaps (amd64 & arm64) for each release of EdgeX to the Snap Store. In contrast to docker deployment, all core,
-security, support, and runtime dependencies are provided in a single snap called [edgexfoundry](https://snapcraft.io/edgexfoundry).
-Additional snaps are available for [App Service Configurable](https://snapcraft.io/edgex-app-service-configurable), as well as the
-standard set of EdgeX reference device services (see [list](#device-service-snaps) below). The edgexfoundry snap also includes Device Virtual to allow users
-to experiment with EdgeX without installing additional snaps.
+## Installing the `edgexfoundry` snap
+[![](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-white.svg){: style="height:4em"}](https://snapcraft.io/edgexfoundry)
```

Comment:
This may look correct when it's been parsed and turned into the documentation file, but when this page is simply previewed in Github, then the style information does not seem to render correctly. Instead, we get a page-wide image and the {style} bit as text after it. We could remove this additional style information and simply use  the official image at:
```
[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-white.svg)](https://snapcraft.io/edgexfoundry)
```

Reply:
Good find. I tried to get the svg from store but the source URL wasn't easy to find. Strangely, the store only provides the code for the badge for HTML and not markdown (I remember it also had markdown before, but maybe I'm wrong).